lib/manager/terragrunt-version/extract.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { getName } from '../../../test/util';
+import { extractPackageFile } from './extract';
+
+describe(getName(__filename), () => {
+  describe('extractPackageFile()', () => {
+    it('returns a result', () => {
+      const res = extractPackageFile('12.0.0\n');
+      expect(res.deps).toMatchSnapshot();
+    });
+    it('skips non ranges', () => {
+      const res = extractPackageFile('latest');
+      expect(res.deps).toMatchSnapshot();
+    });
```

Comment:
This test doesn't work as expected, it will return a dep with `currentValue=latest`.

So remove it or fix the implementation.

Reply:
Opting to remove.